apache-arrow.yaml

```diff
@@ -87,8 +87,7 @@ pipeline:
         -DCMAKE_BUILD_TYPE=MinSizeRel \
         -DCMAKE_INSTALL_LIBDIR=lib \
         -DCMAKE_INSTALL_PREFIX=/usr \
-        -DGLOG_INCLUDE_DIR=/usr/include/glog \
-        -DGLOG_LIB=/usr/lib/libglog.so.1 \
+        -DARROW_USE_GLOG:BOOL=OFF \
```

Comment:
Can we use this patch https://github.com/apache/arrow/pull/40230 instead of turning off the glog

Reply:
Done, can you approve the CI?